lib/init/config.js

```diff
@@ -342,6 +342,9 @@ function assignMongooseModels(options) {
         Object.assign({}, modelModule.scheme, schemeDefaults.mongoose(modelName)),
         modelModule.options
       );
+      if (modelModule.coreDataOptions.uniqueCompound) {
```

Comment:
if coreDataOptions is undefined, this will throw an except, add that to the conditional check

Reply:
done!